app/graphql/mutations/base_mutation.rb

```diff
@@ -0,0 +1,6 @@
+module Mutations
+  # This class is used as a parent for all mutations, and it is the place to have common utilities 
+  class BaseMutation < GraphQL::Schema::Mutation
+    null false
```

Comment:
what does `null false` do?

Reply:
deleted the 2`.keep` files, they must have been generated a command I used from this tutorial!